mailman.cfg

```diff
@@ -0,0 +1,21 @@
+# Please set these as explained in README.md
+SERVE_FROM_DOMAIN=""
+MAILMAN_ADMIN_USER=""
+MAILMAN_ADMIN_EMAIL=""
+
+# only use URL safe characters for the secrets, else you will break at least the DATABASE_URL
+# i.e. do not use blanks ; / ? : @ = &  " < > # % { } | \ ^ ~ [ ] `
+# you can use a commandline like `openssl rand -hex 24` to generate pseudo-random values for these secrets
+HYPERKITTY_API_KEY=""
+POSTGRES_PASSWORD=""
+DJANGO_SECRET_KEY=""
+
+# you might want to change the timezone to your local one
+TZ="UTC"
+
+# likely, you don't need to change these:
+DATABASE_URL_SCHEMA=postgresql
+DATABASE_TYPE="postgres"
+DATABASE_CLASS="mailman.database.postgresql.PostgreSQLDatabase"
+POSTGRES_DB="mailmandb"
+POSTGRES_USER="mailman"
```

Comment:
Would it be possible to add the missing line break here?

Also Docker Compose interprets values behind the `=` equal sign literally, that means the double quotation marks `"` must be avoided.

Reply:
Ups, that's right. Done.